primitives/src/parachain.rs

```diff
@@ -361,7 +412,7 @@ substrate_client::decl_runtime_apis! {
 		/// Get the current duty roster.
 		fn duty_roster() -> DutyRoster;
 		/// Get the currently active parachains.
-		fn active_parachains() -> Vec<Id>;
+		fn active_parachains() -> Vec<(Id, Option<CollatorId>)>;
```

Comment:
```suggestion
		fn active_parachains() -> Parachains;
```

same as per previous comment https://github.com/paritytech/polkadot/pull/341/files#r308987015